src/toolbar/toolbar-controller.js

```diff
@@ -6,11 +6,21 @@
             this.$timeout = $timeout;
             this.currentWindowService = currentWindowService;
             this.maximised = false;
+            this.summarised = false;
             currentWindowService.ready(this.onReady.bind(this));
         }
 
         onReady() {
             this.window = this.currentWindowService.getCurrentWindow();
+            this.window.getBounds(function(bounds) {
+                if (bounds.width === 230) {
+                    this.summarised = true;
+                    this.window.summarised = true;
+                }
+                else {
+                    this.window.summarised = false;
+                }
```

Comment:
Yeah, I'll change that. This originally did more conditional stuff, but the if/else is unnecessary now.